### ISSUE TITLE
Integration test for pathology pipelines

### ISSUE THREAD

**Issue by drbeh** [Project-MONAI/MONAI#5311]

**Comment by KumoLiu** [comment]
Hi @vikashg, I noticed that you closed many tickets without comments. Perhaps you could leave a message informing us of the reason for the closure. For some recent requirements, you can also ask the author if they must be retained and if any arrangements are in place. BTW, would be better to start with older tickets.
Thanks!

**Comment by vikashg** [comment]
@KumoLiu YEs I am trying to write comments wherever possible. This one I didnt write anything as @wyli closed it on Feb 22 right ? Please feel free to reopen

**Comment by vikashg** [comment]
Thanks @drbeh
